include/dd/Package.hpp

```diff
@@ -1954,11 +2002,11 @@ namespace dd {
         }
 
         void printVector(const vEdge& e) {
-            unsigned long long element = 2u << e.p->v;
-            for (unsigned long long i = 0; i < element; i++) {
+            auto element = 2ULL << e.p->v;
```

Comment:
Can be `const`. Maybe make type explicit to avoid confusion of the resulting type of the left shift (`unsigned long long` in this case)?

Reply:
Unfortunately, these methods cannot be made `const` due to `getValueByPath` not being `const` (it uses cached complex numbers).